source/redirects.txt

```diff
@@ -208,14 +208,14 @@
 "docs/software/wpilib-tools/pathweaver/index.rst" "docs/software/pathplanning/pathweaver/index.rst"
 "docs/software/wpilib-tools/pathweaver/integrating-robot-program.rst" "docs/software/pathplanning/pathweaver/integrating-robot-program.rst"
 "docs/software/wpilib-tools/pathweaver/introduction.rst" "docs/software/pathplanning/pathweaver/introduction.rst"
-"docs/software/wpilib-tools/robot-characterization/analyzing-data.rst" "docs/software/pathplanning/robot-characterization/analyzing-data.rst"
```

Comment:
You can't change the existing redirects. Add new redirects at the bottom. The redirects will be parsed through so that they work correctly even with multiple redirects.

Reply:
Yeah i realized this after the fact, am gonna redo them at the bottom